src/component/component/index.js

```diff
@@ -137,19 +137,31 @@ export class Component extends BaseComponent {
         // Register all of the drivers for instantiating components. The model used is -- there's a standard javascript
         // way of rendering a component, then each other technology (e.g. react) needs to hook into that interface.
         // This makes us a little more pluggable and loosely coupled.
-
+        this.driverResults = {};
```

Comment:
Then just confirming that with the approach of returning the module from register method plus setting ```component.driverResults.angular2``` or ```component.angular2``` to get an accessor in automatic mode  approach we have two ways of accessing driver results. While if we stick to always setting ```component.driverResults.angular2``` or ```component.angular2``` its one way that work both cases.